_config.yml

```diff
@@ -0,0 +1,48 @@
+title: Alfresco Helm Charts Documentation
+description: User documentation for the Alfresco component Helm Charts
+theme: just-the-docs
+color_scheme: alfresco
+
+url: https://alfresco.github.io/alfresco-helm-charts
+
+aux_links:
+  Docs: https://docs.alfresco.com/
+  GitHub Repository: https://github.com/Alfresco/alfresco-helm-charts
+
+aux_links_new_tab: true
+
+gh_edit_link: true
+gh_edit_link_text: "Improve this page on GitHub"
```

Comment:
This link redirects to https://github.com/Alfresco/alfresco-helm-charts/edit/main/docs/index.md which doesn't exist.

Reply:
yet, it's just because I renamed that file and is not yet on main